src/sentry/seer/similarity/utils.py

```diff
@@ -13,7 +13,7 @@
 MAX_FRAME_COUNT = 30
 MAX_EXCEPTION_COUNT = 30
 FULLY_MINIFIED_STACKTRACE_MAX_FRAME_COUNT = 20
-SEER_ELIGIBLE_PLATFORMS_EVENTS = frozenset(["python", "javascript", "node", "ruby"])
+SEER_ELIGIBLE_PLATFORMS_EVENTS = frozenset(["python", "javascript", "node", "ruby", "php"])
 SEER_ELIGIBLE_PLATFORMS = frozenset(
```

Comment:
do we need to modify `SEER_ELIGIBLE_PLATFORMS` as well?

Reply:
I added it.